keps/prod-readiness/sig-cli/859.yaml

```diff
@@ -1,3 +1,5 @@
 kep-number: 859
 alpha:
   approver: "@deads2k"
+beta:
+  approver: "@deads2k"
```

Comment:
@johnbelamaric volunteered in https://github.com/kubernetes/enhancements/issues/859#issuecomment-839286332